### ISSUE TITLE
Malignant cells in pleural fluid

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10642]
**Preferred term label:**
Malignant cells in pleural fluid

**Synonyms**
https://uts.nlm.nih.gov/uts/umls/concept/C0080032

thoracentesis

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Various forms of lung cancer and various forms of breast cancer

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001078
- label: Malignant pleural effusion
- parents: [Abnormal pleura morphology (HP:0002103)](https://hpo.jax.org/app/browse/term/HP:0002103)
- pmid: PMID:27246596
- def: The accumulation of a significant amount of exudate in the pleural space, accompanied by the presence of malignant cells or tumour tissue.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10642
- synonyms: Malignant cells in pleural fluid(NONE)
